.circleci/config.yml

```diff
@@ -30,24 +31,107 @@ jobs:
           key: gem-cache-{{ checksum "Gemfile.lock" }}
           paths:
             - vendor/bundle
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - "vendor/bundle"
+  install_node_deps:
+    executor: my-executor
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - npm-cache-{{ checksum "package-lock.json" }}
       - run:
           name: Install node dependencies
           command: npm install
-      - run:
-          name: Build USWDS if needed
-          command: npm run build-uswds
       - save_cache:
           key: npm-cache-{{ checksum "package-lock.json" }}
           paths:
             - node_modules
+  build:
+    executor: my-executor
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - restore_cache:
+          keys:
+            - npm-cache-{{ checksum "package-lock.json" }}
+      - restore_cache:
+          keys:
+            - gem-cache-{{ checksum "Gemfile.lock" }}
+            - gem-cache
+      - run:
+          name: Build USWDS if needed
+          command: npm run build-uswds
+      - run:
+          name: Build site assets
+          command: npm run build-all-assets
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - "_site"
+            - "assets"
+            - "node_modules"
```

Comment:
We're saving the site build, the built assets (CSS/JS), and `node_modules` (required for pa11y and USWDS html-templates)